server/service.go

```diff
@@ -319,6 +319,17 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 		return
 	}
 
+	// If genesisTime and slotTimeSec are set, log how late into the slot the request started
+	if config.GenesisTime > 0 && config.SlotTimeSec > 0 {
+		slotStartTimestamp := config.GenesisTime + (int64(_slot) * config.SlotTimeSec)
+		msIntoSlot := time.Now().UTC().UnixMilli() - (slotStartTimestamp * 1000)
+		log = log.WithField("msIntoSlot", msIntoSlot)
+		log.WithFields(logrus.Fields{
+			"genesisTime": config.GenesisTime,
+			"slotTimeSec": config.SlotTimeSec,
+		}).Infof("getHeader request start at %d milliseconds into slot %d", msIntoSlot, _slot)
```

Comment:
this is likely a usability thing; and it's already done so lets just keep it

but we could also just log the `time.Now().UTC().UniMilli()` and the `_slot` and do the relevant conversions in analysis later -- means less code to keep/maintain/ensure is correct here

Reply:
nah, why add a manual computation step -- for debugging it's way better to log it all together in this line!